torch/package/_package_utils.py

```diff
@@ -0,0 +1,60 @@
+import re
```

Comment:
`_package_utils.py` -> `_file_structure_printer.py` 

It's not a good idea to name file utils. In the limit you end up with things like `utils/utils.py` with every function defined in it (I've literally just be working with a repo like this).